src/Android/Xamarin.Android/Samples/Layers/DisplayAnnotation/DisplayAnnotation.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+
+namespace ArcGISRuntimeXamarin.Samples.DisplayAnnotation
+{
+    [Activity(ConfigurationChanges = Android.Content.PM.ConfigChanges.Orientation | Android.Content.PM.ConfigChanges.ScreenSize)]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Display annotation",
+        "Layers",
+        "Display annotation from a feature service URL.",
+        "")]
+    [ArcGISRuntime.Samples.Shared.Attributes.OfflineData()]
```

Comment:
I recommend removing this attribute - I think its an issue with the sample generator script that includes it even when you don't specify any offline data items